### DISCUSSION TITLE
Making modifications without refreshing the page ?

### DISCUSSION THREAD

**Discussion by TomAtTeam** [GrapesJS/grapesjs#3962] [Q&A]
Hi everyone,

I have a question about GrapesJS with MJML.

When I go on this page : https://grapesjs.com/demo-mjml.html,

Let’s say I’d like to change the color of the title « My Company » and make it red.

It’s working fine, the title turns to red.

But then if I want to put « My Company » in bold it doesn’t work. If I wan’t to make it work I have to refresh the page and then to put it in bold.

Does someone have an idea to make it work without refresh the page please and without using the style manager ?

Thank you !

Thomas

**Comment by ronaldohoch** [comment]
I tested it now and it worked to me 🤔 
![image](https://user-images.githubusercontent.com/2287371/142622741-df2b13b0-eb6b-4354-8d54-6ed8a263fe86.png)

**Reply by TomAtTeam** [reply]
Did you successfully do it with the toolbar (the one on the screen below)
![Capture d’écran (238)](https://user-images.githubusercontent.com/94383921/142629266-e4e10aed-a118-4f9c-a582-567c535dca21.png)
 ?

**Reply by ronaldohoch** [reply]
Hmm, no i didn't, i've made with the right toolbar...

**Comment by aliibrahim123** [comment]
maybe you can use
```
editor.select(editor.getWrapper().replaceWith(editor.getWrapper().getEl().innerHTML));
```

**Reply by TomAtTeam** [reply]
Thank you very much for your answer ! I tried it and it didn't worked. Maybe (and surely) it's because I'm doing it wrong. 

I've made a quick video of my problem :

https://user-images.githubusercontent.com/94383921/142850212-6d1236be-c5de-497f-b8c3-52da52583b60.mp4

Thomas

**Reply by Ju99ernaut** [reply]
For grapesjs-mjml I think you should avoid this because it may lose some component data

**Reply by Ju99ernaut** [reply]
For you to bolden text using the rte you have to select it first.

**Reply by TomAtTeam** [reply]
Thank you for your reply ! It seems to work properly on the demo page (but not in my code so I'm still looking for what's wrong !).
